src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -260,15 +260,36 @@ object DemuxFastqs {
       |    --metadata SampleSheet.csv --metrics metrics.txt --output output_folder
       |```
       |
-      |## Illumina FASTQ File Naming
+      |## Output Standards
       |
-      |The output FASTQs will be written according to Illumina naming standards when the `--illumina-standards` option
-      |is specified.  This requires that the input fastq files contain Illumina format read names with comments containing
-      |`<ReadNum>:<FilterFlag>:<0>:<SampleNumber>` (no angle brackets).  The output create output files with Illumina
-      |names:
+      |The following options affect the output format:
       |
-      |1. The file extension will be `_R1_001.fastq.gz` for read one, and `_R2_001.fastq.gz` for read two (if paired end).
-      |2. The per-sample output prefix will be `<SampleName>_S<SampleOrdinal>_L<LaneNumber>` (without angle brackets).
+      |1. If `--fastq-skip-read-numbers` is specified, then trailing /1 and /2 for R1 and R2 respectively, will not be
+      |appended to e FASTQ read name.  By default they will be appended.
+      |2. If `--fastq-include-sample-barcodes` is specified, then sample barcode will replace the last field in the
+      |first comment in the FASTQ header.
+      |3. If `--illumina-file-names` is specified, the output files will be named according to the Illumina FASTQ file
+      |naming conventions:
+      |
+      |  a. The file extension will be `_R1_001.fastq.gz` for read one, and `_R2_001.fastq.gz` for read two (if paired end).
+      |  b. The per-sample output prefix will be `<SampleName>_S<SampleOrdinal>_L<LaneNumber>` (without angle brackets).
+      |
+      |Options (1) and (2) require the FASTQ read names to contain the following elements:
+      |
+      |`@<instrument>:<run number>:<flowcell ID>:<lane>:<tile>:<x-pos>:<y-pos>:<UMI> <read>:<is filtered>:<control number>:<index>`
+      |
+      |[See the Illumina FASTQ conventions for more details.](https://support.illumina.com/help/BaseSpace_OLH_009008/Content/Source/Informatics/BS/FASTQFiles_Intro_swBS.htm)
+      |
+      |The `--illumina-standards` option may not be specified with the three options above.  Use this option if you
+      |intend to upload to Illumina BaseSpace.  This option implies:
+      |
+      |`--fastq-skip-read-numbers=true --fastq-include-sample-barcodes=false --illumina-file-names=true`
+      |
+      |[See the Illumina Basespace standards described here](https://help.basespace.illumina.com/articles/tutorials/upload-data-using-web-uploader/).
+      |
+      |To output with Illumina conventions that match `bcl2fastq` and `BCLconvert`, use:
+      |
+      |`--fastq-skip-read-numbers=true --fastq-include-sample-barcodes=true --illumina-file-names=true`
```

Comment:
I broke apart the options to control various file and read name/header standards.    Hopefully it's much clearer.